src/components/info-card/info-card.tsx

```diff
@@ -0,0 +1,60 @@
+import { ReactFormState } from "../../redux/models";
+import styles from "./info-card.module.css";
+
+interface InfoCardProps {
+  formData: ReactFormState;
+  isFormFilled: boolean;
+  base64?: string | ArrayBuffer | null;
+  isFormUpdate: boolean;
+}
+
+const InfoCard = (props: InfoCardProps) => {
```

Comment:
```suggestion
const InfoCard = (props: InfoCardProps) => {
const InfoCard = ({ formData, isFormFilled, base64, isFormUpdate }: InfoCardProps) => {
```
Don't forget about _Destructuring assignment_